src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md

```diff
@@ -1,42 +1,132 @@
 
-Scheduled tasks are defined by adding a `cron.json` file at the root of your application's source code.
+### Limitations
 
-The file will be read at the next deployment and the tasks will be launched automatically.
+The following limitations currently exist, be it for security, performance or
+stability reasons:
 
-It must be a **valid JSON file** in the format specified below.
+- The Scalingo Scheduler has been designed to run short-lived tasks.
+  Consequently, **scheduled tasks won't run more than 15 minutes**. They are
+  automatically killed as soon as they reach this limit.
+
+- Moreover, a one-off container started by the Scalingo Scheduler will not run
+  longer than its scheduling interval. For example, a scheduled task set up to
+  run every 10 minutes will be terminated after running for 10 minutes. A
+  scheduled task set to run every 30 minutes will be killed after running for
+  15 minutes (see above).
+
+- The smallest scheduling interval is set to 10 minutes. For example, the
+  Scalingo Scheduler won't let you set up a task to run every 5 minutes (the
+  deployment will fail).
+
+- **A maximum of 5 tasks** can be scheduled through the Scalingo Scheduler.
+  Please get in touch with the support if you need more, they will study your
+  request and advise you.
+
+- Task execution is expected but **not guaranteed**. The Scalingo Scheduler is
+  known to occasionally (but rarely) miss the execution of a scheduled job.
+
+- Execution time can be delayed by a maximum of 10% of the interval between 2
+  tasks (up to 20 minutes).
+
+  * `*/10 * * * *` (every 10 minutes): task will be executed every 10 to 11
+    minutes;
+  * `0 */1 * * *` (every hour): task will be executed between minute 0 and
+    minute 6 of each hour;
+  * `0 0 * * *`: (everyday at midnight): task will be executed between 00:00
+    and 00:20 UTC.
+
+- Two one-off containers running the same scheduled task may overlap for a
+  brief time if the task is not finished when a new one is started.
+
+{% note %}
+All these limitations can be circumvented by using a
+[custom clock process]({% post_url platform/app/task-scheduling/2000-01-01-custom-clock-processes %}).
+{% endnote %}
+
+### Costs
+
+Although the Scalingo Scheduler itself is free, the one-off containers in which
+the commands are run are billed like any other
+[one-off container]({% post_url platform/app/2000-01-01-tasks %}).
+
+Consequently, billing depends on the type of container you defined in your task
+(M is the default container size) and on the job lifespan.
+
+For example, if your job runs during 5 minutes, you will be billed 5 minutes of
+an M container.
+
+## Working with the Scalingo Scheduler
+
+### Enabling the Scalingo Scheduler
+
+The Scalingo Scheduler is enabled automatically as soon as a `cron.json` file
+is present at the root of your application.
+
+### Disabling the Scalingo Scheduler
+
+For now, it's not possible to disable scheduled tasks once a `cron.json` has
+been committed.
```

Comment:
I'm not fond of this phrasing. If one remove the `cron.json`, we don't execute anymore the tasks, don't we?

Reply:
New suggestion of wording in https://github.com/Scalingo/documentation/pull/1915/commits/22a7de15789981fbcc5fd0eeacd80f1081d22d66